esy-lib/ChildProcess.ml

```diff
@@ -67,7 +67,7 @@ let withProcess ?(env=`CurrentEnv) ?(resolveProgramInEnv=false) ?stdin ?stdout ?
           resolveCmdInEnv ~env prg
         | _ -> Ok prg
       in
-      return (prg, Array.of_list (prg::args))
+      return ("", Array.of_list (prg::args))
```

Comment:
@andreypopp - I did some research on this here: https://ocsigen.org/lwt/3.2.1/api/Lwt_process

On Windows, we need to pass in `""` to the first parameter, so that the executable (in this case, `node`) gets picked up via the `PATH`.

On Linux/OSX, if we pass in `""` to the first parameter, the first argument gets used - the only difference is that it falls back to the `PATH` if its not found. I believe this behavior is acceptable but wanted to check. Seems to work fine from testing on OSX.

Reply:
Looks ok, there are couple of places which use `Lwt_process` directly in `esyi`, we should fix them too soon.